manifests/composer.pp

```diff
@@ -32,10 +32,17 @@
     command => "curl -sS -o ${composer_path} ${composer_url}",
     unless  => "[ -f ${composer_path} ] && [ \"`md5 -q ${composer_path}`\" = \"${composer_checksum}\" ]",
     cwd     => $php::config::root,
+    loglevel => 'debug',
```

Comment:
https://github.com/boxen/puppet-php/pull/97/commits/7d34f6bb8b28f7d742b587aa26d9d13e8d36c580 notes that it's to suppress the download-php-composer notice. https://docs.puppet.com/puppet/latest/types/exec.html#exec-attribute-logoutput shows that loglevel defaults to a value which will only print the result on failure (or when the return value does not match the 'returns' parameter (which for curl would be anything non zero).

This might have been added on for personal preference but I think we could probably just revert this line back to how it was prior to the PR and if users start to observe excessive notices from composer downloads we can look to reinstate it.